### DISCUSSION TITLE
When is the next release expected (or where can we see the roadmap)

### DISCUSSION THREAD

**Discussion by nestorsokil** [argoproj/argo-rollouts#1333] [General]
Pretty much explained in the title. I really think a public roadmap even a very simple one would be helpful to the community.
Thanks 🙌

**Comment by nextrevision** [comment]
There are two published roadmaps from what I can see:

https://argoproj.github.io/argo-rollouts/roadmap/
https://github.com/argoproj/argo-rollouts/milestones
